STM32Resources/BuildEnvironment_Setup/BuildEnvironment_Setup.adoc

```diff
@@ -0,0 +1,81 @@
+
+
+= *Setting up and Using an STM32 Build Environment on Linux*
+A guide for setting up a linux system to build projects for STM32
+
+
+This guide assumes you are starting with a Windows 64 bit system and intend to use a virtual machine in Oracle Virtualbox. I installed Linux Mint 18.3 If you already have a linux system set up, are using a Mac, or want to set it up a diffeent way figure out what parts of this guide you have already completed yourself and rejoin at a later step.
```

Comment:
Good asciidoc practice is having one sentence per line. And in general, documentation should be written in third person without personal anectodes. For example this excerpt would be better written as,


```asciidoc
This guide assumes the base operating system is 64 bit Windows.
The steps include setting up a virtual machine in Oracle VirtualBox, but if a Linux system is already set up in any fashion, this step can be skipped.
The Linux system that this guide has been written with is Linux Mint 18.3.
```